crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -83,5 +93,31 @@ def clusters_healthy?
     def compute_resources_available?
       Api::Crowbar.new.compute_resources_available?
     end
+
+    def prepare_nodes_for_crowbar_upgrade_background
+      @thread = Thread.new do
+        Rails.logger.debug(I18n.t("api.upgrade.prepare.started_background"))
+        status = prepare_nodes_for_crowbar_upgrade
```

Comment:
why is this useless when I use it at the end of the method as a return value? otherwise the thread will be returned